### DISCUSSION TITLE
Ranking System

### DISCUSSION THREAD

**Discussion by TheAdventurer64** [organicmaps/organicmaps#3056] [Q&A]
When you look at your OSM account in MAPS.ME, there is a "Position in the ranking" system that doesn't work anymore. Considering how this feature was removed during the migration to Organic Maps, why was this removed and is it possible to get it back? ![image](https://user-images.githubusercontent.com/50059322/181835687-cca55b98-cfac-4110-86d6-b07b41e5bf2b.jpeg)

**Comment by biodranik** [comment]
That was a custom server on mapsme side that ranked only mapsme users. We didn't want to make our life harder by developing and supporting it. It would be better to use some existing OSM-based service, or OSM server itself.

**Reply by hb0nd** [reply]
perhaps https://hdyc.neis-one.org/
